runelite-client/src/main/java/net/runelite/client/plugins/clanchat/ClanChatPlugin.java

```diff
@@ -668,4 +678,28 @@ private void confirmKickPlayer(final String kickPlayerName)
 			.option("2. Cancel", Runnables::doNothing)
 			.build();
 	}
+
+	private void colorIgnoredPlayers()
+	{
+		Widget clanChatList = client.getWidget(WidgetInfo.CLAN_CHAT_LIST);
+		if (clanChatList == null || clanChatList.getChildren() == null)
+		{
+			return;
+		}
+
+		Color ignoreColor = config.showIgnores() ? config.showIgnoresColor() : Color.WHITE;
+		NameableContainer<Ignore> ignoreContainer = client.getIgnoreContainer();
+		// Iterate every 3 widgets, since the order of widgets is name, world, icon
+		// and World \\d+ could be a players name
```

Comment:
this comment is just explaining why your previous implementation is a bad idea, but has nothing to do with this. i would remove it.